package.json

```diff
@@ -25,6 +25,8 @@
     "@angular/platform-browser": "^18.2.0",
     "@angular/platform-browser-dynamic": "^18.2.0",
     "@angular/router": "^18.2.0",
+    "@lmdb/lmdb-win32-x64": "^3.1.3",
```

Comment:
Can you please remove these additional packages?
Because of them the builds are failing.
Please avoid committing package.json & package-lock.json

Reply:
yes, working on it

Reply:
Done.
Please review.

Reply:
I think the package-lock.json still exists in the commit, can you please revert it ?

Reply:
The diff is a closing brace. And I can't find what the difference is.

Reply:
Alright !